### PR TITLE
add time elapsed as an input, fix wrong order

### PR DESCRIPTION
**Pull request by jack-song**

### CODE REVIEW

File: showHistory/typedUrls.js [hunk 1]
```diff
@@ -124,80 +124,141 @@ document.addEventListener('DOMContentLoaded', function () {
...
+        } else {
+          return " - ";
+        }
+      }
+
+      function normalizeInterval(start, end) {
+        // 1000*60*60 = 3600000 ms in an hour
+        var elapsed = end-start;
+
+        if(elapsed >= 0 && elapsed <= 3600000) {
```
Comment (kimyousee) [inline showHistory/typedUrls.js]: 3600000 should be a constant

File: showHistory/typedUrls.js [hunk 2]
```diff
@@ -124,80 +124,141 @@ document.addEventListener('DOMContentLoaded', function () {
...
         } else {
           return 0;
         }
       }
 
+      function getName(index) {
+        if(index in indexToSite) {
+          return indexToSite[index];
+        } else {
+          return " - ";
```
Comment (kimyousee) [inline showHistory/typedUrls.js]: why

**Review by kimyousee — APPROVED** [review]
yay new variable names

File: showHistory/typedUrls.js (hunk 2 above)
Comment (jack-song) [reply showHistory/typedUrls.js]: just an error case, no site could be found

File: showHistory/typedUrls.js (hunk 1 above)
Comment (jack-song) [reply showHistory/typedUrls.js]: okaaaaay

**Pull request merged**